app/controllers/generic_phrases_controller.rb

```diff
@@ -1,14 +1,67 @@
 class GenericPhrasesController < ApplicationController
+  include Searchable
+
   def index
-    @items = [
-      {
-        verb: "to find",
-        adjective: "information",
-      },
-      {
-        verb: "acquire somthing",
-        adjective: "delivery",
-      },
-    ]
+    options = {
+      page: page,
+      sort_key: sort_key,
+      sort_direction: sort_dir,
+      verb: verb,
+      adjective: adjective,
+    }
+
+    @presenter = GenericPhrasesPresenter.new(generic_phrase_results, verb_results, adjective_results, search_params, options)
+  end
+
+private
+
+  def generic_phrase_results
+    options = {
+      sort_key: sort_key,
+      sort_dir: sort_dir,
+    }
+
+    options[:verb] = verb if verb.present?
+    options[:adjective] = adjective if adjective.present?
+
+    GenericPhrase.search(from_date_as_datetime, to_date_as_datetime, options)
+  end
+
+  def verb_results
+    Verb.unique_sorted.map(&:name)
+  end
+
+  def adjective_results
+    Adjective.unique_sorted.map(&:name)
+  end
+
+  def sort_key
```

Comment:
I have a mental note that we need to move key, page stuff etc to `sortable` but that's a future thing

Reply:
I based the sort presenter I have off of the one in Content Data, would be good to think about how we do it.